### PR TITLE
Sassify/clean-up anim.scss

### PR DESCRIPTION
**Pull request by JoannesJ**
Create sass mixin for defining animated sprite emotes
Create sass mixin for defining emote css animations
Remove duplicate animations
Enable "animate forever" for DJPepo
Fix autoprefixer not working on imported scss modules
Remove vendor prefixes from anim.scss
Remove flash and spin animations.

### CODE REVIEW

File: assets/anim.scss [hunk 1]
```diff
@@ -1,20 +1,235 @@
...
-}
+@import 'common';
+
+$emote-prefix: '.chat-emote-';
+$css-animated-emote-animations: ();
+$context-css-animated-emote-mixin: false;
+
+// Mixin for defining an emote with a css animation.
+// Should be used in combination with emote-animation mixin.
+@mixin css-animated-emote($name, $can-animate-forever: false) {
```
Comment (KenanY) [inline assets/anim.scss]: I wonder if `$can-animate-forever` should default to `true`?

Comment (JoannesJ) [reply assets/anim.scss]: Currently DJPepo is the only css animated emote that uses it.
I would stick to what is the most common configuration.

Comment (KenanY) [reply assets/anim.scss]: Before this patch, all animated PNG emotes had the `&-animate-forever` selector. Now only DJPepo will?

Comment (JoannesJ) [reply assets/anim.scss]: No, the animated PNGs(defined by sprite-animation mixin) all animate per default.
PJPepo uses css animation, not a sprite. 
So it uses the same mixin as stuff like "OverRustle" and "DANKMEMES".

I would split those categories up in to 2 files, honestly.
And put the remaining animations with the classes that use them.
But I left that for another day.

Comment (KenanY) [reply assets/anim.scss]: Ah, missed that mixin.

**Comment by KenanY** [comment]
Landed as https://github.com/MemeLabs/chat-gui/commit/b3f76cfdde5241f298bc4ba284163a331206b6a7. Thanks!

**Pull request closed without merging**